### PR TITLE
Confirm page small adjustments.

### PR DESCRIPTION
**Pull request by nibivi77**
### Description
Changes requested by Greg from a previous PR that was merged accidentally.

### CODE REVIEW

**Review by gregory-j-baker — DISMISSED** [review]

File: frontend/public/locales/en/apply.json [hunk 1]
```diff
@@ -73,11 +73,11 @@
   "confirm": {
     "page-title": "Confirmation",
     "keep-copy": "Keep a copy of your application",
-    "print-copy-text": "You will <bold>not</bold> be receiving a <strong>confirmation email</strong>. It is important to keep a copy of your application. You can use this button to print.",
+    "print-copy-text": "You will <bold>not</bold> be receiving a <bold>confirmation email</bold>. It is important to keep a copy of your application. You can use this button to print.",
```
Comment (sebastien-comeau) [inline frontend/public/locales/en/apply.json:76]: `<b>` instead of `<bold>` since `bold` is not a valid HTML tag

Comment (sebastien-comeau) [reply frontend/public/locales/en/apply.json:76]: https://www.w3schools.com/html/html_formatting.asp#:~:text=HTML%20%3Cb%3E%20and%20%3Cstrong%3E%20Elements

Comment (nibivi77) [reply frontend/public/locales/en/apply.json:76]: <bold> is not a literal HTML tag. Its a component place holder used with <Trans>
`<Trans ns={i18nNamespaces} i18nKey="confirm.print-copy-text" components={{ bold: <strong /> }} />`

Comment (sebastien-comeau) [reply frontend/public/locales/en/apply.json:79]: Gotcha. `strong` is supported by `Trans` component so you shouldn't have to do it this way. Look at `transKeepBasicHtmlNodesFor` config https://react.i18next.com/latest/trans-component.

Comment (nibivi77) [reply frontend/public/locales/en/apply.json:79]: Oh! Very nice!

**Pull request merged**